run_robot_test.sh

```diff
@@ -168,6 +168,13 @@ case "$(uname -s)" in
 esac
 
 
-./venv/bin/robot  -d "${TEST_ARTIFACT_DIR}" -x xunit_test_result.xml -r test_report.html ${TEST_VARIABLES} --variablefile ${TEST_VARIABLES_FILE} ${EXTRA_ROBOT_ARGS} ${TEST_CASE_FILE}
+./venv/bin/robot  ${EXTRA_ROBOT_ARGS} -d ${TEST_ARTIFACT_DIR} -x xunit_test_result.xml -r test_report.html ${TEST_VARIABLES} --variablefile "${TEST_VARIABLES_FILE}" "${TEST_CASE_FILE}"
+
+#Re-Run failed tests and merge the outputs
+if [[ $? -ne 0 ]] && [[ -f "${TEST_ARTIFACT_DIR}/output.xml" ]]; then
+    ./venv/bin/robot  ${EXTRA_ROBOT_ARGS} -d ${TEST_ARTIFACT_DIR} -x xunit_test_result.xml -r test_report.html ${TEST_VARIABLES} --variablefile "${TEST_VARIABLES_FILE}" --rerunfailed "${TEST_ARTIFACT_DIR}/output.xml" --output "output_rerun.xml" "${TEST_CASE_FILE}"
+
+    ./venv/bin/rebot -d ${TEST_ARTIFACT_DIR} -o "output.xml" -x xunit_test_result.xml -r test_report.html --merge "${TEST_ARTIFACT_DIR}/output.xml" "${TEST_ARTIFACT_DIR}/output_rerun.xml"
```

Comment:
Please attach the test results of successful execution with rerun passed cases

Reply:
The logs are already attached, did I miss something in there?

Reply:
I am good with logs. I was looking for report, xunit and output xml files

Reply:
Thanks Rajan, attached the complete results directory